File: BackEnd/ExamManagement/DPD.HR.Business.Dto/Dto/TicketLaborEquipmentDetailDto.cs
namespace DPD.HumanResources.Dtos.Dto;

public class TicketLaborEquipmentDetailDto
{
    public Guid Id {get;set;}
    public Guid IdTicketLaborEquipment {get;set;}
    public Guid IdCategoryLaborEquipment {get;set;}
    public double? Quantity {get;set;}
    public DateTime? CreatedDate {get;set;}
    public int? Status {get;set;}
    public Guid IdEmployee { get; set; }
    public string? EquipmentCode { get; set; }
    public bool IsCheck { get; set; }
}

File: FrontEnd/src/models/ticketLaborEquipmentDetailDTO.ts
export interface TicketLaborEquipmentDetailDTO {
  id: string;
  idTicketLaborEquipment: string;
  idCategoryLaborEquipment: string | null;
  quantity: number;
  createdDate: Date;
  status: number | null;
  idEmployee: string | null;
  equipmentCode: string | null;
  isCheck: boolean;
}
